crates/evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -125,7 +125,18 @@ pub fn apply<DB: DatabaseExt>(
                     Err(err) => return Some(Err(err)),
                 }
             }
-            Ok(urls.encode().into())
+            Ok(DynSolValue::Array(
+                urls.into_iter()
+                    .map(|t| {
+                        DynSolValue::FixedArray(vec![
+                            DynSolValue::String(t[0].clone()),
+                            DynSolValue::String(t[1].clone()),
+                        ])
+                    })
+                    .collect_vec(),
+            )
+            .abi_encode()
```

Comment:
cc @DaniPopes this shoudl be possible to do w/o having to do the iter map collect stuff

Reply:
https://github.com/foundry-rs/foundry/pull/5928#discussion_r1346568383